README.md

```diff
@@ -49,14 +49,14 @@ macOS        | <img src="stores_presence/macos-about.png" /> | <img src="stores_
       - [References](#references-1)
     - [Building for Android on macOS](#building-for-android-on-macos)
       - [Reference](#reference)
-  - [Building for iOS](#building-for-ios)
-    - [iOS build issues and solution (or workarounds)](#ios-build-issues-and-solution-or-workarounds)
+  - [Building for iOS](docs/build_ios.md) (seperate file in docs)
```

Comment:
You need to submit only changes described in the issue:
- create a dedicated branch
- only add the required changes
- create a pull request from that branch

Reply:
@costa-rica on your side you can:
- git pull latest version of original master branch
- create a branch from here
- only add the required changes of this navigation menu image fix
- create a pull request from that branch

Reply:
this MR can be just closed to avoid confusion